### PR TITLE
feat: add drizzle orm with sqlite

### PR DESCRIPTION
**Pull request by aaa006bd**

### CODE REVIEW

**Comment by anbraten** [comment]
@aaa006bd Could you save the users on login to the database so drizzle is directly used.

**Comment by aaa006bd** [comment]
> @aaa006bd Could you save the users on login to the database so drizzle is directly used.

sure. This PR will close 2 issues then.

**Comment by anbraten** [comment]
That's fine.

**Comment by aaa006bd** [comment]
@anbraten for some reason my app isn't starting anymore. can you check if it works in your system?

**Comment by anbraten** [comment]
@aaa006bd Could you resolve conflicts

File: server/api/github/callback.ts [hunk 1]
```diff
@@ -14,11 +18,33 @@ export default defineEventHandler(async (event) => {
...
+    method: 'POST',
+    body: {
+      client_id: config.public.github.clientId,
+      client_secret: config.github.clientSecret,
+      code,
+    },
+  });
   if (response.error) {
     return sendRedirect(event, '/');
+    return sendRedirect(event, '/');
```
Comment (anbraten) [inline server/api/github/callback.ts:31]: @aaa006bd Why return twice?

File: server/api/github/callback.ts [hunk 2]
```diff
@@ -14,11 +18,33 @@ export default defineEventHandler(async (event) => {
...
+    loginName: user.data.login,
+    name: user.data.name,
+    avatarUrl: user.data.avatar_url,
+    email: user.data.email,
+  }).run();
+
+  // TODO: set cookie using jwt-token etc instead of plain token
+  setCookie(event, 'gh_token', token, { path: '/' });
+
+  return sendRedirect(event, '/');
```
Comment (anbraten) [inline server/api/github/callback.ts:48]: Second return?

File: composables/github.ts [hunk 3]
```diff
@@ -23,11 +23,27 @@ export const fetchGithubUser = async () => {
...
+    method: 'POST',
+    body:{
+      id:user.id,
+      name: user.name,
+      loginName: user.login,
+      avatarUrl : user.avatar_url,
+      email: user.email
+    }
+  })
+}
```
Comment (anbraten) [inline composables/github.ts:49]: Not used

File: server/api/github/callback.ts [hunk 4]
```diff
@@ -14,11 +18,33 @@ export default defineEventHandler(async (event) => {
...
     },
   });
+  const response: any = await $fetch('https://github.com/login/oauth/access_token', {
+    method: 'POST',
+    body: {
+      client_id: config.public.github.clientId,
+      client_secret: config.github.clientSecret,
+      code,
+    },
+  });
```
Comment (anbraten) [inline server/api/github/callback.ts:28]: duplicated

Comment (aaa006bd) [reply server/api/github/callback.ts]: @anbraten yeah I saw. This new merge editor in vs code is very hard to use. I am fixing it.

**Review by anbraten — APPROVED** [review]

**Pull request merged**